libs/git/info.go

```diff
@@ -86,7 +86,7 @@ func fetchRepositoryInfoAPI(ctx context.Context, path string, w *databricks.Work
 		result.CurrentBranch = gi.Branch
 		result.WorktreeRoot = fixedPath
 	} else {
-		log.Warnf(ctx, "Failed to load git info from %s", apiEndpoint)
+		log.Debugf(ctx, "Failed to load git info from %s", apiEndpoint)
```

Comment:
Infof would also hide it but will show up with -v (once we implement that; the plan for it is to surface Info+).